Kernel/Interrupts/SpuriousInterruptHandler.cpp

```diff
@@ -16,6 +17,22 @@ UNMAP_AFTER_INIT void SpuriousInterruptHandler::initialize(u8 interrupt_number)
     handler->register_interrupt_handler();
 }
 
+void SpuriousInterruptHandler::initialize_for_disabled_master_pic()
+{
+    auto* handler = new SpuriousInterruptHandler(7);
+    // Note: When we disable the PIC, the IRQ base offset is set to 0x20
+    register_generic_interrupt_handler(0x20 + 7, *handler);
```

Comment:
Can we make these comments useless by making `0x20` a constexpr constant and using it both here and in the PIC code?

Reply:
I rather do that in a different pull request so I don't need to touch that code for now.

Reply:
I don't think a different pull request is appropriate. That constant was only used in one location, which makes hardcoding it ok, but now it's used in multiple locations, so repeating it is an issue. You can create a separate commit that extract the constant into a constexpr/define and then use it in your current commit.

Reply:
I used a constexpr, hopefully it's good enough :)